src/backend/distributed/commands/table.c

```diff
@@ -928,7 +941,11 @@ PreprocessAlterTableStmt(Node *node, const char *alterTableCommand,
 		sqlForTaskList = DeparseTreeNode((Node *) newStmt);
 	}
 
-	ddlJob->commandString = alterTableCommand;
+	ddlJob->commandString = NULL;
```

Comment:
I wouldn't mind if this said `useInitialDDLCommandString ? alterTableCommand : NULL`